src/support/SupportResourceType.ts

```diff
@@ -32,6 +32,15 @@ export const supportResources: SupportResourceType[] = [
     resourceType: 'billing-project',
     loadSupportSummaryFn: (id: FullyQualifiedResourceId) => Ajax().Billing.adminGetProject(id.resourceId),
   },
-  { displayName: 'Dataset', resourceType: 'dataset', loadSupportSummaryFn: undefined },
-  { displayName: 'Snapshot', resourceType: 'datasnapshot', loadSupportSummaryFn: undefined },
+  {
+    displayName: 'Dataset',
+    resourceType: 'dataset',
+    loadSupportSummaryFn: (id: FullyQualifiedResourceId) => Ajax().DataRepo.admin().adminRetrieveDataset(id.resourceId),
```

Comment:
Another way to do this would be make `adminRetrieveDataset` accept `FullyQualifiedResourceId` directly, to avoid creating an extra lambda here.
```
export interface DataRepoContract {
  admin: () => {
    adminRetrieveSnapshot: (snapshot: FullyQualifiedResourceId) => Promise<Snapshot>;
...

export const DataRepo = (signal?: AbortSignal): DataRepoContract => ({
  admin: () => {
    return {
      adminRetrieveSnapshot: async (snapshot: FullyQualifiedResourceId) =>
        callDataRepo(`admin/v1/snapshots/${snapshot.resourceId}`, signal),
...

  {
    displayName: 'Dataset',
    resourceType: 'dataset',
    loadSupportSummaryFn: Ajax().DataRepo.admin().adminRetrieveDataset,
  },
```

Reply:
I kind of like the way it is currently, just because it is consistent with the other endpoints and then the ajax call has everything it needs and nothing more. Thoughts on keeping it?